JavaScript/Task_Day1/Basic_Mathematical.js

```diff
@@ -0,0 +1,14 @@
+//www.codewars.com/kata/57356c55867b9b7a60000bd7/train/javascript
+//Mathematical_Operations
+
+function basicOp(operation, value1, value2) {
+  if (operation == "+") {
```

Comment:
You can use switch case, instead of several if else chains

Reply:
Made the changes. Switch case used instead of else if ladder.